tasks/test.rake

```diff
@@ -4,17 +4,21 @@ require 'rspec/core/rake_task'
 ## spec / unit tests
 ##
 
+vendor_selector = lambda do |spec_file|
+  spec_file.start_with?('vendor/')
+end
+
 desc 'Executes every spec file, one at a time.'
 task 'test:spec:each' do
   failures = []
-  Dir.glob("**/spec/**/*_spec.rb").each do |spec_file|
+  ::Dir.glob('**/spec/**/*_spec.rb').reject(&vendor_selector).each do |spec_file|
```

Comment:
Ditto on `::`

Reply:
Ditto on the explanation.

However, in this place, the `::` is unnecessary; it is just my style.  
I default to absolute addressing and only use relative ones when intentionally wanted to use relative modules.

If you prefer consistency on the module addressing, and relative addressing is your default, I can always change it to be the same. Just ping me with the confirmation.